docs/to_clarify.md

```diff
@@ -0,0 +1,8 @@
+## Question
+1. What are the existing solutions that was mentioned in point 2 of the "Possible Use Cases"
```

Comment:
We could be more specific: What are the "existing solutions and databases" that we need to integrate with as mentioned in Point 2 of "Possible Use Cases"?

Also, this question is somewhat linked to what @alvinlim540 mentioned concerning Booking Engine and the ease of integration; whether there's documentation & assistance will be provided to us for integration. It should be consolidated as 1 big point.

Reply:
end-to-end Digital Concierge (which includes digital checking-in/out):
https://www.youtube.com/watch?v=EAjs3HVGCPU

Unlock room using mobile:
https://www.youtube.com/watch?v=TDlEK3Ca0RY
https://www.youtube.com/watch?v=qHLdz_nA6zA

Reply:
Point to note: This link is the hotel tech award. Can review on the  current technology that internationally what they are using. https://partners.hoteltechreport.com/hotel-tech-awards/2020-winners/

Reply:
There's quite a bit of noise in these articles. For e.g. we aren't concerned with Property Management, Commercial Wi-Fi etc. Can you shortlist what exactly should we be looking at?

Reply:
For Check in Check Out Process: 
Quore  https://www.quore.com/
Canary Technologies https://www.canarytechnologies.com/ 
ASSA ABLOY Global Solutions https://campaigns.assaabloyglobalsolutions.com/en-contactlesshospitality?gclid=Cj0KCQjwvIT5BRCqARIsAAwwD-SnuK5vsGg0NV51zFu1nK6_9NoX3RtJ3jriKu83d5TcsTKeY6a0rfAaAsY9EALw_wcB
INTELITY https://intelity.com/

For concierge Process: 
ALICE (2020 Winner) https://www.aliceplatform.com/
SuitePad https://www.suitepad.de/en/
Crave https://crave-emenu.com/hotel-tablets/
Volo https://www.getvolo.com/
Whistle https://www.trywhistle.com/
Bookboost https://www.bookboost.io/
Volara https://volara.io/

Reply:
Good links. I can't find Check-in/out for Quore though.

Reply:
Removed.